components/input/input-group.component.ts

```diff
@@ -56,6 +56,7 @@ export class NzInputGroupWhitSuffixOrPrefixDirective {
       <span
         *ngIf="isAffix; else contentTemplate"
         class="ant-input-affix-wrapper"
+        [attr.disabled]="disabled || null"
```

Comment:
It's better to use `ant-input-affix-wrapper-disabled` instead of `disabled`, ng-zorro-antd should follow `ant-design` :)

Reply:
ok